Sdl.Community.Toolkit.LanguagePlatform/Sdl.Community.Toolkit.LanguagePlatform.csproj

```diff
@@ -57,6 +57,7 @@
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Web" />
```

Comment:
We should remove reference to System.Web. This seems to be needed for Html encode/decode which is also part of System.Net - https://docs.microsoft.com/en-us/dotnet/api/system.net.webutility.htmldecode?view=netframework-4.7.2